src/pages/workspace/WorkspaceMembersPage.tsx

```diff
@@ -382,26 +385,80 @@ function WorkspaceMembersPage({policyMembers, personalDetails, route, policy, se
         </View>
     );
 
+    const handleBulkAction = (action: WorkspaceMemberBulkActionType) => {
+        // TODO: implement make member and make admin
+        switch (action) {
+            case CONST.POLICY.MEMBERS_BULK_ACTION_TYPES.REMOVE:
+                askForConfirmationToRemove();
+                break;
+            case CONST.POLICY.MEMBERS_BULK_ACTION_TYPES.MAKE_MEMBER:
+            case CONST.POLICY.MEMBERS_BULK_ACTION_TYPES.MAKE_ADMIN:
+            default:
+                break;
+        }
+    };
+
+    const getBulkActionsButtonOptions = () => {
+        const iconSettings = {
+            iconWidth: 40,
+            iconHeight: 40,
+        };
+        const options: Array<DropdownOption<WorkspaceMemberBulkActionType>> = [
+            {
+                text: translate('workspace.people.removeMembersTitle'),
+                value: CONST.POLICY.MEMBERS_BULK_ACTION_TYPES.REMOVE,
+                icon: Expensicons.RemoveMembers,
+                onSelected: () => handleBulkAction(CONST.POLICY.MEMBERS_BULK_ACTION_TYPES.REMOVE),
+                ...iconSettings,
+            },
+        ];
+
+        if (selectedEmployees.find((employee) => policyMembers?.[employee].role === CONST.POLICY.ROLE.ADMIN)) {
+            options.push({
+                text: translate('workspace.people.makeMember'),
+                value: CONST.POLICY.MEMBERS_BULK_ACTION_TYPES.MAKE_MEMBER,
+                icon: Expensicons.MakeMember,
+                onSelected: () => handleBulkAction(CONST.POLICY.MEMBERS_BULK_ACTION_TYPES.MAKE_MEMBER),
```

Comment:
Same here and for the other options, let's just directly call the specific method

Reply:
👍

Reply:
adressed